tests/test_basic.py

```diff
@@ -1,8 +1,9 @@
 import pytest
 from analyticsdf.analyticsdataframe import AnalyticsDataframe
+
 import numpy as np
 
-# initiate an AnalyticsDataframe with default name
+## Initiate an AnalyticsDataframe with default name
```

Comment:
Any coding standard for changing the number of hash?

Reply:
It's just a personal coding habit BUT I know what you mean and my apologies for editing your code without submitting a PR!!
However, I do think we should standardize all the comments/function descriptions/file descriptions at some point in the future. But that's not the priority right now so let's leave that till later.